docs/proposals/TEMPLATE.md

```diff
@@ -17,47 +17,47 @@ Agent itself.
 ## Problem
 
 *If necessary, add a more detailed sketch of the problem. Add any notes,
-pictures, text or details that illuminate why this is a problem we need to
-solve. Keep it as concise as possible.*
+pictures, text, or details that illuminate why this is a problem Datadog Agent 
+integrations community needs to solve. Keep it as concise as possible.*
 
 ## Constraints
 
 *If necessary, note any constraints or requirements that any solution must
 fulfill. For example:*
 
-    1. We have to target Python 2.7.
+    1. Datadog Agent has to target Python 2.7.
     2. Memory footprint should not increase more than 10%.
     3. Every check in `integrations-core` and `integrations-extra` should be
        supported.
 
 ## Recommended Solution
 
 *Describe your solution in the bare minimum detail to be understood. Explain
-why it's better than what we have and better than other options. Address
+why it's better than what Datadog Agent has and better than other options. Address
```

Comment:
not sure here about to who the `we` is referring to

Reply:
"what we have" means "the current implementation" of whatever part of the repo

Reply:
changed

Reply:
Is that better ?
```suggestion
why it's better than what's already in place and better than other options. Address
```